sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/documentanalysis/administration/DocumentModelAdministrationClientTestBase.java

```diff
@@ -91,7 +91,7 @@ static TokenCredential getCredentialByAuthority(String endpoint) {
         }
     }
 
-    static void validateCopyAuthorizationResult(CopyAuthorization actualResult) {
+    static void validateCopyAuthorizationResult(DocumentModelCopyAuthorization actualResult) {
```

Comment:
Should the name for copy auth include target somewhere? We call it `TargetAuthorization` in Python

Reply:
It would then be called `TargetDocumentModelCopyAuthorization `.
I think it is fine with what we have since all the properties named within have `target` prefix and also we haven't had any customer issues on this so don't mind keeping it as is for now.